Makefile.include

```diff
@@ -30,8 +30,8 @@ include $(RIOTBASE)/Makefile.docker
 # Static code analysis tools provided by LLVM
 include $(RIOTBASE)/Makefile.scan-build
 
-# Path to the current directory relative to the git root
-BUILDRELPATH ?= $(shell git rev-parse --show-prefix)
+# Path to the current directory relative to RIOTBASE
+BUILDRELPATH ?= ${PWD:${RIOTBASE}/%=%}/
```

Comment:
I'm not sure Make understands bash'isms like that. To iterate over alternatives you can use `$(or)`:

```make
$(or ${var1},${var2})
$(or ${var1},$(or ${var2},${var3}))
$(or ${var1},$(or ${var2},$(of ${var3},${var4})))
```